.github/workflows/lint.yaml

```diff
@@ -43,13 +43,43 @@ jobs:
         run: |
           git remote add upstream https://github.com/coreruleset/coreruleset
           git fetch --tags upstream
-
+          git fetch upstream main
+          echo ${{ github.base_ref }}
+          if [ -n "${{ github.base_ref }}" ]; then
```

Comment:
Is there any chance that variable `${{ github.base_ref }}` is empty? What happens if that's it?

Reply:
Yes, in which case `version_flag` will be empty.